services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/docker.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, HTTPException, Response, status
+
+from ..core.docker_utils import supports_volumes_with_quota
+
+router = APIRouter()
+
+
+@router.post(
```

Comment:
Sorry, but this is a really strange entrypoint? 
- it does not identify any resource
- ``supported`` is not an operation (i.e. custom operation are suppose to "replace" the http operation POST)
- you use the 404 (not found) to respond that quotas are not supported! 

Some suggestions? You want to run a check to determine whether volumes with are supported or not
- user an existing API resource as path ``docker/volumes`` 
- create an operation e.g. ``check`` or ``analyze`` , i.e. ``POST docker/volumes:check`` 
- respond with a body that has resulting information e.g.
```json
{
  "are_quotas_supported" : false
}
```
- This in addition is reusable for other properties you want to check/analyze about volumes
- Use http error codes to indicate states like it could not be "checked" (e.g. if docker client fails I would use a 503 for it)